ccmlib/node.py

```diff
@@ -85,6 +88,124 @@ def __decode(self, value):
 _active_tasks_pattern = re.compile(r'\s*([\w-]+)\s+\w+\s+(?P<ks>\w+)\s+(?P<cf>\w+)\s+\d+\s+\d+\s+\w+\s+\d+\.\d+%')
 
 
+class RepairOptions:
+    class KeyspaceReplication(Enum):
```

Comment:
Le'ts drop the KeyspaceReplication limiter. It is not very useful for the test but introduces a lot of confusing.